consensus/polybft/blockchain_wrapper.go

```diff
@@ -26,7 +26,7 @@ type blockchainBackend interface {
 	CurrentHeader() *types.Header
 
 	// CommitBlock commits a block to the chain.
-	CommitBlock(block *types.Block) ([]*types.Receipt, error)
+	CommitBlock(block *types.FullBlock) error
```

Comment:
What I would do here is just use remove `State` field from the `StateBlock` and use it, because it is the same as `FullBlock` (or other way around: remove `StateBlock` and use `FullBlock` everywhere where needed instead).

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1063/commits/8f1ed3b53aeeb4a3ee5f23c309c954501bd526fa